src/modules/public/configs/sagas/configsFetchSaga.ts

```diff
@@ -15,6 +15,6 @@ export function* configsFetchSaga() {
         yield put(setBlocklistStatus({ status: 'allowed' }));
     } catch (error) {
         yield put(configsError(error));
-        yield put(alertPush({message: error.message, code: error.code, type: 'error'}));
+        yield put(sendError(error, 'alert'));
```

Comment:
Would be better to have universal method for error handler. Example for this case:
`
yield all(handleError(error)));
`
Use `all` method, since `handleError` can send many actions inside

Reply:
Okay